src/main/java/com/kirinpatel/sync/util/FileDownloader.java

```diff
@@ -0,0 +1,23 @@
+package com.kirinpatel.sync.util;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+
+public class FileDownloader {

```

Comment:
Maybe just make this a private method in dependency verifier

Reply:
Done.